src/modules/users/data/queries.ts

```diff
@@ -367,13 +367,12 @@ export const getUserColonyTransactions: Query<
       events: { Transfer },
```

Comment:
There's a comment on line `350` that references `blocksBack`, but it's all good :)

https://github.com/JoinColony/colonyDapp/blob/33029846ca73bfe616c1330663e0132c73ba3afb/src/modules/users/data/queries.ts#L348-L351

Reply:
Ah nice catch! I've removed that